tests/_test_openapi.py

```diff
@@ -7,11 +7,12 @@
 from reasoner_pydantic import components
 
 TAG = "v1.2"
-response = httpx.get(f"https://raw.githubusercontent.com/NCATSTranslator/ReasonerAPI/{TAG}/TranslatorReasonerAPI.yaml")
-reference_schemas = yaml.load(
-    response.text,
-    Loader=yaml.FullLoader,
-)["components"]["schemas"]
+response = httpx.get(
+    f"https://raw.githubusercontent.com/NCATSTranslator/ReasonerAPI/{TAG}/TranslatorReasonerAPI.yaml"
+)
+reference_schemas = yaml.load(response.text, Loader=yaml.FullLoader,)[
+    "components"
+]["schemas"]
```

Comment:
Sometimes black formatting is the worst. I am fine with this for the simplicity of applying and enforcing style, but voicing my discontent.